### DISCUSSION TITLE
How to get currently playing song

### DISCUSSION THREAD

**Discussion by pr13260** [sigma67/ytmusicapi#395] [Q&A]
i tried reading documentation but there seems to be no mention of it ;(

**Comment by sigma67** [comment]
This is not something that YouTube Music supports. You can try `get_watch_playlist` or `get_song`

**Reply by IrgendwerUsingGitHub** [reply]
Do you know the reason why YouTube doesn't support it? Is it a decision on purpose that they dont want to provide this feature or is it possible that it will come in the near future?

**Reply by sigma67** [reply]
Ask Google. This is just my hobby project
